src/yoroi-wallets/utils/validators.test.ts

```diff
@@ -0,0 +1,25 @@
+import {YoroiNFTModerationStatus} from '../types'
+import {isValidModerationStatus} from './validators'
+
+describe('isValidModerationStatus', () => {
+  it('should return true on valid NFT Moderation status', () => {
```

Comment:
can we leverage string template to improve readabilitiuy

Reply:
I've added `const parseModerationStatus = (status: unknown): YoroiNFTModerationStatus | undefined` and `type YoroiNFTModerationStatus = 'yellow' | 'red' | 'green' | 'pending' | 'manual_review'`